webpack/assets/javascripts/react_app/components/Layout/layout.scss

```diff
@@ -29,45 +30,68 @@
   }
 }
 
-#react-app-root > .nav-pf-vertical {
-  background-color: $nav-pf-vertical-secondary-active-bg-color;
+#react-app-root {
+  .react-container {
+    margin-right: auto;
+    margin-left: 200px;
+    padding-left: 20px;
+    padding-right: 20px;
 
-  .list-group {
-    .list-group-item {
-      a {
-        span {
-          color: $nav-pf-vertical-icon-color;
-        }
+    &.collapsed-nav {
+      margin-left: 75px;
+    }
+  }
 
-        &:hover {
-          background-color: $nav-pf-vertical-active-bg-color;
+  /*
+    patternfly-sass hides the vertical-nav on smaller screens, see: https://github.com/patternfly/patternfly-sass/blob/5e7cce3445d5b1af1e16500695b9f33edc6a4f6c/assets/javascripts/patternfly-functions.js#L260
+    the react-container needs to be shifted when it happens.
+  */
+  @media only screen and (max-width: $pf-global--breakpoint--md) {
+    .react-container {
+      margin-left: 0;
+    }
+  }
```

Comment:
This is getting overridden by the previous rule:
` &.collapsed-nav {
         margin-left: 75px;
}
`

Reply:
done, fixed the situation where screen was small and nav collapsed :+1: